static/js/actions.js

```diff
@@ -41,6 +41,7 @@ $(function() {
         //zakładki
         $('.zakladki-content article').hide();
         $('.zakladki-content article:first').show();
+        $('ul.tabs > li:first').addClass('active');
```

Comment:
Też zastanawiałem się nad tym. Można usunąć i ustawiać w kodzie HTML-a odpowiednią zakładkę, ale póki nie dowiemy się o tym, że gdzieś ma być inaczej, to może lepiej zostawić tak jak teraz jest.